lib/matplotlib/tests/test_collections.py

```diff
@@ -325,6 +325,19 @@ def test__EventCollection__set_linestyle():
     splt.set_title('EventCollection: set_linestyle')
 
 
+@image_comparison(baseline_images=['EventCollection_plot__set_ls_dash'],
```

Comment:
Just a question: does this need to test all of the backends, or can it target specific ones? e.g., is a png test sufficient?